bricks.module

```diff
@@ -155,7 +155,8 @@ function _bricks_tree_items($element, $items) {
  */
 function bricks_process_field(&$vars) {
   $element = &$vars['element'];
-  if ($element['#field_name'] == 'field_body') {
+  $field = field_info_field($element['#field_name']);
```

Comment:
In this case, we are loading field info for every processing field, but it's loaded from cache, so I don't think it will cause any performance issues